support/config/precached.conf

```diff
@@ -167,6 +168,7 @@ program_blacklist = [
   "/usr/bin/pidof",
   "/usr/bin/prename",
   "/usr/bin/ps",
+  "/usr/bin/python*",
```

Comment:
I don't want to blacklist python

Reply:
for python, java, perl, lua... I think it makes more sense to follow the binary called after these languages and not start at the language itself.
Currently precachedtop shows that it is watching python. Well I have 3 apps started with python ... . I did find out what precached was interested in. Maybe just start after the language binary to profile.

Also these are languages with their own command prompt/terminal/shell. Do not preload my random commands(file which may get accessed) I may be running in the language shells.